toolkit/tools/imagecustomizerapi/overlay.go

```diff
@@ -0,0 +1,70 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+package imagecustomizerapi
+
+import (
+	"fmt"
+	"strings"
+)
+
+type Overlay struct {
+	LowerDir  string           `yaml:"LowerDir"`
+	UpperDir  string           `yaml:"UpperDir"`
+	WorkDir   string           `yaml:"WorkDir"`
+	Partition *VerityPartition `yaml:"Partition"`
```

Comment:
It seems a little odd that the type is called `VerityPartition` and is used here. Should that type be renamed to something else?

Reply:
Fixed.